README.md

```diff
@@ -52,15 +52,15 @@ Pick a tag to jump to, open it with `:tag`
 
 Pick a tag to jump to, open it with `:stag`
 
-### `PickCommand(choice_command, pick_args, vim_command)`
+### `PickCommand(choice_command, pick_args, vim_command, escapable_output)`
 
 Run the `choice_command` in from the shell and pipe the results to pick run with
 `pick_args` and call the `vim_command` with the selected choice.
 
 For example, you could implement `PickFile()` like this:
 
 ```viml
-call PickCommand("find * -type f", "", ":edit")
+call PickCommand("find * -type f", "", ":edit", "")
```

Comment:
Could you swap this for an integer, such as the actual code: https://github.com/calleerlandsson/pick.vim/commit/6b6a6ff6f748fc0b3bffb869419bb3333d59dd84

Thanks!

Reply:
sure